src/niswitch/examples/niswitch_get_device_info.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python
+
+import argparse
+import niswitch
+import sys
+
+parser = argparse.ArgumentParser(description='Gets device information from a specified switch module.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-n', '--name', default='PXI1Slot2', help='Resource name of a National Instruments Switch.')
+parser.add_argument('-ri', '--relay_information', default=False, action='store_true', help='Provides relay name, position, and count for all relays on a switch')
```

Comment:
remove option, always get relay info?

Reply:
The issue is that relay info can be huge, and it causes command line to expand and you then cannot see the device info.